libs/eo/shared/ui-page-templates/src/lib/footer/eo-footer.component.ts

```diff
@@ -75,6 +76,7 @@ const selector = 'eo-footer';
         alt="Energinet"
         class="${selector}__img watt-space-stack-l"
       />
+      <ng-content></ng-content>
```

Comment:
suggestion: We could have a component test demonstrating the content projection slot.